### ISSUE TITLE
Install virt-manager on Mac Majave failed

### ISSUE THREAD

**Issue by netmanyys** [jeffreywildman/homebrew-virt-manager#163]
Does anyone meet this issue before? i tried to reinstall the command line tool with reboot but no luck.

```
bash-3.2$ brew install virt-manager virt-viewer
Updating Homebrew...
==> Installing virt-manager from jeffreywildman/virt-manager
==> Downloading https://files.pythonhosted.org/packages/c6/3e/d00f1500aa0e8a69323101c33f6e6910bbc68d34df3e8a0b1e510219a956/virtualenv-20.2.2.tar.gz
Already downloaded: /Users/y0y00rc/Library/Caches/Homebrew/downloads/7bd9cf07bf9c64150432db355d70d87afc2c9bdbf522a1366db32125bf41fe80--virtualenv-20.2.2.tar.gz
==> Downloading https://files.pythonhosted.org/packages/d7/d8/05696357e0311f5b5c316d7b95f46c669dd9c15aaeecbb48c7d0aeb88c40/appdirs-1.4.4.tar.gz
Already downloaded: /Users/y0y00rc/Library/Caches/Homebrew/downloads/d46628d315f103ea527907020cf5f6c94365bbb549783f75a794526482afb672--appdirs-1.4.4.tar.gz
==> Downloading https://files.pythonhosted.org/packages/2f/83/1eba07997b8ba58d92b3e51445d5bf36f9fba9cb8166bcae99b9c3464841/distlib-0.3.1.zip
Already downloaded: /Users/y0y00rc/Library/Caches/Homebrew/downloads/b25916e85753e80fccf86f2f1ba088b86d1a82c93cd36bdb72613c81cb830c55--distlib-0.3.1.zip
==> Downloading https://files.pythonhosted.org/packages/14/ec/6ee2168387ce0154632f856d5cc5592328e9cf93127c5c9aeca92c8c16cb/filelock-3.0.12.tar.gz
Already downloaded: /Users/y0y00rc/Library/Caches/Homebrew/downloads/6132211d03cc3113b135a5b72510e3b6ea1df5bf527eb89d71c3f45085c252a7--filelock-3.0.12.tar.gz
==> Downloading https://files.pythonhosted.org/packages/6b/34/415834bfdafca3c5f451532e8a8d9ba89a21c9743a0c59fbd0205c7f9426/six-1.15.0.tar.gz
Already downloaded: /Users/y0y00rc/Library/Caches/Homebrew/downloads/e99698a932bde9f5fb8ca552923ae339984f284e1c7764df95c12175f573c0d7--six-1.15.0.tar.gz
==> Downloading https://libvirt.org/sources/python/libvirt-python-5.10.0.tar.gz
Already downloaded: /Users/y0y00rc/Library/Caches/Homebrew/downloads/b76fb5c149eddbeee27380e0bef764856ce56622788f0a5547fda4853e4922b8--libvirt-python-5.10.0.tar.gz
==> Downloading https://pypi.io/packages/source/i/idna/idna-2.8.tar.gz
Already downloaded: /Users/y0y00rc/Library/Caches/Homebrew/downloads/89cd3c9503f30cf4794067fbb3bb85881a04d4a79002cd52dd006617f4dee6d5--idna-2.8.tar.gz
==> Downloading https://pypi.io/packages/source/c/certifi/certifi-2019.11.28.tar.gz
Already downloaded: /Users/y0y00rc/Library/Caches/Homebrew/downloads/d305888203f80870f04cbff5b615679be127a718ffeec3b3ce5db16bdc4abfa2--certifi-2019.11.28.tar.gz
==> Downloading https://pypi.io/packages/source/c/chardet/chardet-3.0.4.tar.gz
Already downloaded: /Users/y0y00rc/Library/Caches/Homebrew/downloads/0b520e43abac95fdb4182899f49c1c7ced69c67c4bc132b3801691b993eb33f9--chardet-3.0.4.tar.gz
==> Downloading https://pypi.io/packages/source/u/urllib3/urllib3-1.25.7.tar.gz
Already downloaded: /Users/y0y00rc/Library/Caches/Homebrew/downloads/04dc027e49125d42a130fd408cde60c9d9a87ce021683a3e49bf067a62316261--urllib3-1.25.7.tar.gz
==> Downloading https://pypi.io/packages/source/r/requests/requests-2.22.0.tar.gz
Already downloaded: /Users/y0y00rc/Library/Caches/Homebrew/downloads/4253f7dff3c4f4a5a7fe142af689d95ed2e77979f696451a78d96412e90b0f94--requests-2.22.0.tar.gz
==> Downloading https://virt-manager.org/download/sources/virt-manager/virt-manager-2.2.1.tar.gz
Already downloaded: /Users/y0y00rc/Library/Caches/Homebrew/downloads/77e22463a35b2e2bde8d02253c10129aabb1e5dfbfe2e6f33ef87bd35955dcf2--virt-manager-2.2.1.tar.gz
Warning: A newer Command Line Tools release is available.
Update them from Software Update in System Preferences or run:
  softwareupdate --all --install --force

If that doesn't show you an update run:
  sudo rm -rf /Library/Developer/CommandLineTools
  sudo xcode-select --install

Alternatively, manually download them from:
  https://developer.apple.com/download/more/.

==> Patching
==> python3 -c import setuptools... --no-user-cfg install --prefix=/private/tmp/virt-manager--homebrew-virtualenv-20201218-81108-8ahqsl/target/vendor --install-scripts
==> python3 -c import setuptools... --no-user-cfg install --prefix=/private/tmp/virt-manager--homebrew-virtualenv-20201218-81108-8ahqsl/target/vendor --install-scripts
==> python3 -c import setuptools... --no-user-cfg install --prefix=/private/tmp/virt-manager--homebrew-virtualenv-20201218-81108-8ahqsl/target/vendor --install-scripts
==> python3 -c import setuptools... --no-user-cfg install --prefix=/private/tmp/virt-manager--homebrew-virtualenv-20201218-81108-8ahqsl/target/vendor --install-scripts
==> python3 -c import setuptools... --no-user-cfg install --prefix=/private/tmp/virt-manager--homebrew-virtualenv-20201218-81108-8ahqsl/target --install-scripts=/priva
==> python3 -s /private/tmp/virt-manager--homebrew-virtualenv-20201218-81108-8ahqsl/target/bin/virtualenv -p python3 /usr/local/Cellar/virt-manager/2.2.1_3/libexec
==> /usr/local/Cellar/virt-manager/2.2.1_3/libexec/bin/pip install -v --no-deps --no-binary :all: --ignore-installed /private/tmp/virt-manager--libvirt-python-20201218
Last 15 lines from /Users/y0y00rc/Library/Logs/Homebrew/virt-manager/07.pip:
  Found link https://files.pythonhosted.org/packages/0b/f5/be8e741434a4bf4ce5dbc235aa28ed0666178ea8986ddc10d035023744e6/pip-20.2.4.tar.gz#sha256=85c99a857ea0fb0aedf23833d9be5c40cf253fe24443f0829c7b472e23c364a1 (from https://pypi.org/simple/pip/) (requires-python:>=2.7,!=3.0.*,!=3.1.*,!=3.2.*,!=3.3.*,!=3.4.*), version: 20.2.4
  Found link https://files.pythonhosted.org/packages/fb/46/26d13ba147ba430f9cda0d0cf599a041d142a5c8b1a90ff845ebce7fba0f/pip-20.3b1-py2.py3-none-any.whl#sha256=122fcd82deac1153c1699f29815bfab3f876e5bbe018cc2df1297f9802572a97 (from https://pypi.org/simple/pip/) (requires-python:>=2.7,!=3.0.*,!=3.1.*,!=3.2.*,!=3.3.*,!=3.4.*), version: 20.3b1
  Found link https://files.pythonhosted.org/packages/7f/61/2da3c027ad7bd4bc87a3ee7e7160c93e7500dac3536e02ff93008e9b3460/pip-20.3b1.tar.gz#sha256=819c710a5c8d8c5e44695d03e51cb23b08c070e1ae6a5d6910a89e248e0ff29c (from https://pypi.org/simple/pip/) (requires-python:>=2.7,!=3.0.*,!=3.1.*,!=3.2.*,!=3.3.*,!=3.4.*), version: 20.3b1
  Found link https://files.pythonhosted.org/packages/55/73/bce122d1ed0217b3c1a3439ab16dfa94bbeabd0d31755fcf907493abf39b/pip-20.3-py2.py3-none-any.whl#sha256=3236fe7288d155c238bb6c85d3e784db3a8e592e827b83fea4d36d8b749ecc4b (from https://pypi.org/simple/pip/) (requires-python:>=2.7,!=3.0.*,!=3.1.*,!=3.2.*,!=3.3.*,!=3.4.*), version: 20.3
  Found link https://files.pythonhosted.org/packages/03/41/6da553f689d530bc2c337d2c496a40dc9c0fdc6481e5df1f3ee3b8574479/pip-20.3.tar.gz#sha256=9ae7ca6656eac22d2a9b49d024fc24e00f68f4c4d4db673d2d9b525c3dea6e0e (from https://pypi.org/simple/pip/) (requires-python:>=2.7,!=3.0.*,!=3.1.*,!=3.2.*,!=3.3.*,!=3.4.*), version: 20.3
  Found link https://files.pythonhosted.org/packages/ab/11/2dc62c5263d9eb322f2f028f7b56cd9d096bb8988fcf82d65fa2e4057afe/pip-20.3.1-py2.py3-none-any.whl#sha256=425e79b20939abbffa7633a91151a882aedc77564d9313e3584eb0416c28c558 (from https://pypi.org/simple/pip/) (requires-python:>=2.7,!=3.0.*,!=3.1.*,!=3.2.*,!=3.3.*,!=3.4.*), version: 20.3.1
  Found link https://files.pythonhosted.org/packages/cb/5f/ae1eb8bda1cde4952bd12e468ab8a254c345a0189402bf1421457577f4f3/pip-20.3.1.tar.gz#sha256=43f7d3811f05db95809d39515a5111dd05994965d870178a4fe10d5482f9d2e2 (from https://pypi.org/simple/pip/) (requires-python:>=2.7,!=3.0.*,!=3.1.*,!=3.2.*,!=3.3.*,!=3.4.*), version: 20.3.1
  Skipping link: yanked for reason: <none given>: https://files.pythonhosted.org/packages/3d/0c/01014c0442830eb38d6baef0932fdcb389279ce74295350ecb9fe09e048a/pip-20.3.2-py2.py3-none-any.whl#sha256=8d779b6a85770bc5f624b5c8d4d922ea2e3cd9ce6ee92aa260f12a9f072477bc (from https://pypi.org/simple/pip/) (requires-python:>=2.7,!=3.0.*,!=3.1.*,!=3.2.*,!=3.3.*,!=3.4.*)
  Skipping link: yanked for reason: <none given>: https://files.pythonhosted.org/packages/51/63/86e147c44335b03055e58a27c791d94fff4baaf08d67852f925ab9b90240/pip-20.3.2.tar.gz#sha256=aa1516c1c8f6f634919cbd8a58fc81432b0fa96f421a97d05a205ee86b07c43d (from https://pypi.org/simple/pip/) (requires-python:>=2.7,!=3.0.*,!=3.1.*,!=3.2.*,!=3.3.*,!=3.4.*)
  Found link https://files.pythonhosted.org/packages/54/eb/4a3642e971f404d69d4f6fa3885559d67562801b99d7592487f1ecc4e017/pip-20.3.3-py2.py3-none-any.whl#sha256=fab098c8a1758295dd9f57413c199f23571e8fde6cc39c22c78c961b4ac6286d (from https://pypi.org/simple/pip/) (requires-python:>=2.7,!=3.0.*,!=3.1.*,!=3.2.*,!=3.3.*,!=3.4.*), version: 20.3.3
  Found link https://files.pythonhosted.org/packages/ca/1e/d91d7aae44d00cd5001957a1473e4e4b7d1d0f072d1af7c34b5899c9ccdf/pip-20.3.3.tar.gz#sha256=79c1ac8a9dccbec8752761cb5a2df833224263ca661477a2a9ed03ddf4e0e3ba (from https://pypi.org/simple/pip/) (requires-python:>=2.7,!=3.0.*,!=3.1.*,!=3.2.*,!=3.3.*,!=3.4.*), version: 20.3.3
Given no hashes to check 155 links for project 'pip': discarding no candidates
WARNING: You are using pip version 20.3.1; however, version 20.3.3 is available.
You should consider upgrading via the '/usr/local/Cellar/virt-manager/2.2.1_3/libexec/bin/python -m pip install --upgrade pip' command.
Removed build tracker: '/private/tmp/pip-req-tracker-krr33pdn'

If reporting this issue please do so at (not Homebrew/brew or Homebrew/core):
  https://github.com/jeffreywildman/homebrew-virt-manager/issues

These open issues may also help:
Error installing virt-manager https://github.com/jeffreywildman/homebrew-virt-manager/issues/145
osx virt-manager console screen is black https://github.com/jeffreywildman/homebrew-virt-manager/issues/113
virt-manager crashing with Python quit unexpectedly while using the libgdk-3.0.dylib https://github.com/jeffreywildman/homebrew-virt-manager/issues/62
```

**Comment by YvanMasterAll** [comment]
The same problem, my osx is catalina, and virt-viewer can be installed normally, but virt-manager cannot.

**Comment by xsited** [comment]
PR #161 
helped me out with this issue

**Comment by netmanyys** [comment]
@xsited  Thank you, yes it is helpful!

` 15:00:04 ~ $ sudo vi /usr/local/Homebrew//Library/Taps/jeffreywildman/homebrew-virt-manager/virt-manager.rb`

**Comment by netmanyys** [comment]
Issue got resolved after i upgrade libvirt-python to 6.1.0, thank you @xsited

**Comment by dmasiero** [comment]
> Issue got resolved after i upgrade libvirt-python to 6.1.0, thank you @xsited
> 
>  15:00:04 ~ $ sudo vi /usr/local/Homebrew//Library/Taps/jeffreywildman/homebrew-virt-manager/virt-manager.rb

For added clarity, edit virt-manager.rb and apply changes outlined in PR #161 👍 Successfully able to install virt-manager after making these changes.
